app/src/services/conditional_token.ts

```diff
@@ -76,6 +76,9 @@ class ConditionalTokenService {
   getBalanceOf = async (ownerAddress: string, positionId: string): Promise<BigNumber> => {
     return this.contract.balanceOf(ownerAddress, positionId)
   }
+  getBalanceOfByBlock = async (ownerAddress: string, positionId: string, block: number): Promise<BigNumber> => {
```

Comment:
If you set `positionId: BigNumber` it works :)
Not sure if it causes any other problems but I don't think it should

Reply:
Nice find!